### PR TITLE
doc: provide THCI file for Thread 1.3 Certification testing

### PR DESCRIPTION
**Pull request by edmont**
Thread Test Harness issues require incompatible APIs to be used in THCI files for Thread 1.3 testing. Providing here the apropriate file for such DUT.

Signed-off-by: Eduardo Montoya <eduardo.montoya@nordicsemi.no>

### CODE REVIEW

**Review by canisLupus1313 — APPROVED** [review]

File: doc/nrf/ug_thread_certification.rst [hunk 1]
```diff
@@ -92,11 +92,13 @@ Complete the following steps to prepare for the certification tests:
...
 #. Prepare Thread Test Harness.
 
-   a. Copy the provided :file:`ncs/modules/lib/openthread/tools/harness-thci/OpenThread.py` file into :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI\\nRF_Connect_SDK.py`.
+   a. Copy the provided :file:`ncs/modules/lib/openthread/tools/harness-thci/OpenThread.py` file into :file:`C:\\GRL\\Thread1.2\\Thread_Harness\\THCI\\nRF_Connect_SDK.py`.
 
-   b. Copy an image of your choice to :file:`C:\\GRL\\Thread1.1\\Web\\images\\nRF_Connect_SDK.jpg`.
+   b. Copy the provided :file:`ncs/nrf/samples/openthread/cli/harness-thci-1-3/nRF_Connect_SDK_1_3.py` file into :file:`C:\\GRL\\Thread1.2\\Thread_Harness\\THCI\\nRF_Connect_SDK_1_3.py`.
 
-   c. Edit :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI\\nRF_Connect_SDK.py` as follows:
+   c. Copy an image of your choice to :file:`C:\\GRL\\Thread1.2\\Web\\images\\nRF_Connect_SDK.jpg` and to :file:`C:\\GRL\\Thread1.2\\Web\\images\\nRF_Connect_SDK_1_3.jpg`.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:99]:
```suggestion
   c. Copy images of your choice to :file:`C:\\GRL\\Thread1.2\\Web\\images\\nRF_Connect_SDK.jpg` and :file:`C:\\GRL\\Thread1.2\\Web\\images\\nRF_Connect_SDK_1_3.jpg`.
   You can use the same image for both.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Comment by canisLupus1313** [comment]
@edmont seems that Linter complains about python 2 style `print`.
``` 
************* Module nRF_Connect_SDK_1_3
samples/openthread/cli/harness-tchi-1-3/nRF_Connect_SDK_1_3.py:3467:9: E0001: Missing parentheses in call to 'print'. Did you mean print(...)? (<unknown>, line 3467) (syntax-error)
Error: Process completed with exit code 1.
```

File: doc/nrf/ug_thread_certification.rst [hunk 2]
```diff
@@ -92,11 +92,13 @@ Complete the following steps to prepare for the certification tests:
...
 #. Prepare Thread Test Harness.
 
-   a. Copy the provided :file:`ncs/modules/lib/openthread/tools/harness-thci/OpenThread.py` file into :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI\\nRF_Connect_SDK.py`.
+   a. Copy the provided :file:`ncs/modules/lib/openthread/tools/harness-thci/OpenThread.py` file into :file:`C:\\GRL\\Thread1.2\\Thread_Harness\\THCI\\nRF_Connect_SDK.py`.
 
-   b. Copy an image of your choice to :file:`C:\\GRL\\Thread1.1\\Web\\images\\nRF_Connect_SDK.jpg`.
+   b. Copy the provided :file:`ncs/nrf/samples/openthread/cli/harness-thci-1-3/nRF_Connect_SDK_1_3.py` file into :file:`C:\\GRL\\Thread1.2\\Thread_Harness\\THCI\\nRF_Connect_SDK_1_3.py`.
 
-   c. Edit :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI\\nRF_Connect_SDK.py` as follows:
+   c. Copy images of your choice to :file:`C:\\GRL\\Thread1.2\\Web\\images\\nRF_Connect_SDK.jpg` and to :file:`C:\\GRL\\Thread1.2\\Web\\images\\nRF_Connect_SDK_1_3.jpg`.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:99]:
```suggestion
   c. Copy images of your choice to :file:`C:\\GRL\\Thread1.2\\Web\\images\\nRF_Connect_SDK.jpg` and :file:`C:\\GRL\\Thread1.2\\Web\\images\\nRF_Connect_SDK_1_3.jpg`.
   You can use the same image for both.
```
Missed the added new line.

Comment (edmont) [reply doc/nrf/ug_thread_certification.rst]: Fixed, thanks.

**Review by wiba-nordic — APPROVED** [review]

File: doc/nrf/ug_thread_certification.rst [hunk 3]
```diff
@@ -92,11 +92,15 @@ Complete the following steps to prepare for the certification tests:
...
-   a. Copy the provided :file:`ncs/modules/lib/openthread/tools/harness-thci/OpenThread.py` file into :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI\\nRF_Connect_SDK.py`.
+   a. Copy the provided :file:`ncs/modules/lib/openthread/tools/harness-thci/OpenThread.py` file into :file:`C:\\GRL\\Thread1.2\\Thread_Harness\\THCI\\nRF_Connect_SDK.py`.
 
-   b. Copy an image of your choice to :file:`C:\\GRL\\Thread1.1\\Web\\images\\nRF_Connect_SDK.jpg`.
+   b. Copy the provided :file:`ncs/nrf/samples/openthread/cli/harness-thci-1-3/nRF_Connect_SDK_1_3.py` file into :file:`C:\\GRL\\Thread1.2\\Thread_Harness\\THCI\\nRF_Connect_SDK_1_3.py`.
 
-   c. Edit :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI\\nRF_Connect_SDK.py` as follows:
+   c. Copy images of your choice to :file:`C:\\GRL\\Thread1.2\\Web\\images\\nRF_Connect_SDK.jpg` and :file:`C:\\GRL\\Thread1.2\\Web\\images\\nRF_Connect_SDK_1_3.jpg`.
+
+   You can use the same image for both.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:101]:
```suggestion
     You can use the same image for both.
```
One more little thing. That was my bad, I gave you the wrong indentation.

**Pull request merged**